### PR TITLE
Feature/beefed up description

### PR DESCRIPTION
**Pull request by mjobin-mdsol**
From my email, after giving a quick presentation of DiceBag at opencode.ca
Here are some small precisions to the README.

I branched off master, so there are two empty merge commits attached to my PR.
Let me know if this is a problem.

### CODE REVIEW

**Comment by asmith-mdsol** [comment]
While you're here, you should add yourself to the list of contributors @mjobin-mdsol!

**Comment by mjobin-mdsol** [comment]
done ! thanks

On Fri, Sep 27, 2013 at 12:10 PM, Andrew Smith notifications@github.comwrote:

> While you're here, you should add yourself to the list of contributors
> @mjobin-mdsol https://github.com/mjobin-mdsol!
> 
> —
> Reply to this email directly or view it on GitHubhttps://github.com/mdsol/dice_bag/pull/66#issuecomment-25256914
> .

## 

_Mathieu Jobin_| Sr. Software Engineer | Medidata Solutions Worldwide
http://www.mdsol.com/ mjobin@mdsol.com | direct: (418) 907 9515

File: README.md [hunk 1]
```diff
@@ -8,6 +8,12 @@ Twelve-Factor App][1]. Configuration values are picked up from the environment
 and used to populate configuration files from templates. Pre-packaged templates
 for common configuration files are provided.
 
+Although Rails already supports ERB syntax for its YML configuration files, DiceBag will generate a final
+static file that will work without keeping your deployment environment variables in sync with your
+production environment variables. For security reasons, these environments may sometimes differ.
+
+Also DiceBag will work with any kind of text files, not only YML. We use it for initializers in Ruby even.
```
Comment (jcarres-mdsol) [inline README.md]: we use it -> for instance can be used

File: README.md [hunk 2]
```diff
@@ -8,6 +8,12 @@ Twelve-Factor App][1]. Configuration values are picked up from the environment
 and used to populate configuration files from templates. Pre-packaged templates
 for common configuration files are provided.
 
+Although Rails already supports ERB syntax for its YML configuration files, DiceBag will generate a final
+static file that will work without keeping your deployment environment variables in sync with your
```
Comment (jcarres-mdsol) [inline README.md:12]: I am not sure of the meaning of this

Comment (mjobin-mdsol) [inline README.md:12]: did you see my email about me presenting Dicebag to a local rails crowd ? 
the immediate reaction was that "rails does support erb in yml files already"

looked it up, and it does indeed.

@asmith-mdsol suggested I add a notes to the README so people understand better the purpose of this.

File: README.md (hunk 1 above)
Comment (mjobin-mdsol) [inline README.md]: fix in f47c445

**Pull request merged**